### ISSUE TITLE
Files in /public folder are not served in production mode

### ISSUE THREAD

**Issue by ranyefet** [fastify/fastify-vite#105]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

4.13.0

### Plugin version

3.0.8

### Node.js version

16.15.1

### Operating system

macOS

### Operating system version (i.e. 20.04, 11.3, 10)

latest

### Description

Hello,

In my Vite application I'm using the [public folder](https://vitejs.dev/guide/assets.html#the-public-directory) feature to serve static files without transformations such as `favicon.svg` and other js/css files.

After building for production and running my fastify app I noticed that the these files are not served.

It appears that `fastify-vite` is only serving `index.html` and everything in the `assets` folder.

```
build
  assets/
    index.[hash].js
  favicon.svg - NOT SERVED
  script1.js - NOT SERVED
  index.html
```

I would expect that it will support public folder feature and serve everything in the `build` folder as is.

### Steps to Reproduce

1. Add `public` folder under your `client` folder
2. Put some file inside, for example `robots.txt` 
3. Run build command
4. Run app in production mode and access `localhost/robots.txt`

### Expected Behavior

Fastify Vite should serve all files in build folder as static files and not just `index.html` and `assets` folder.

**Comment by tantaman** [comment]
workaround for you:

```
import fastifyStatic from "@fastify/static";

await server.register(fastifyStatic, {
  root: new URL("./public", import.meta.url),
});
```

**Comment by ranyefet** [comment]
@tantaman Thanks I already have a workaround but I still think it should be solved in fastidy-vite

**Comment by galvez** [comment]
You're right! This should be handled by `@fastify/vite` — I'm adding it in the next minor.

**Comment by galvez** [comment]
Fixed in `v6.0.1`.
